docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -199,12 +199,16 @@ need to be migrated or deleted, and upgrade the internal indices to the
 +
 You can also call the Elasticsearch migration APIs directly:
 +
-`/_xpack/migration/assistance`:: Runs a series of checks on your cluster,
-nodes, and indices and returns a list of issues that need to be
+`/_xpack/migration/deprecations`:: Retrieves information about cluster-, node-,
+and index-level settings that use deprecated features.
++
+`/_xpack/migration/assistance`:: deprecated[6.7.0] Runs a series of checks on
+your cluster, nodes, and indices and returns a list of issues that need to be
 fixed before you can upgrade to {version}.
```

Comment:
This should probably say something like

```
`/_xpack/migration/assistance`:: deprecated[6.7.0] Returns a list of indices that need to 
be reindexed before you can upgrade to {version}.
```

As it doesn't actually check anything else - the description as written applies more to the `migration/deprecations` API.